src/ray/raylet/worker_pool.cc

```diff
@@ -780,6 +788,20 @@ void WorkerPool::OnWorkerStarted(const std::shared_ptr<WorkerInterface> &worker)
   }
 }
 
+void WorkerPool::ExecuteOnPrestartWorkersStarted(std::function<void()> callback) {
```

Comment:
do we have unit test for this logic in worker_pool_test?

Reply:
Seems unrelated to this PR